resize update (#47)
biometricITC/Fingerprint-Toolbox: FINGERPRINT_Toolbox_Overview.adoc

Fingerprint image acquisition for flat PAIs is the same for all attacks. An image of the subject’s finger is captured using a high quality commercial fingerprint scanner. The image can be from a slap or rolled image. If a rolled image is acquired, the evaluator may want to crop the fingerprint image to make a smaller mold containing only the central part of the fingerprint that would usually be in contact with the TOE sensor during normal operation. This region should be large enough to cover as much of the sensor as possible when presenting the PAI to the TOE. This same finger shall be used to enroll the subject into the TOE. Each image shall be acquired under controlled and favorable conditions so as to produce high quality friction ridge information. The image file is preferred to be in bitmap, TIFF, RAW or other uncompressed file format when received from the scanner.

> The image then needs to be processed such that the resultant image contains only black (0 in the 8-bit grayscale color space) and white (255 in the 8-bit grayscale color space) pixels. This can be done by hand using photo editing software suitable for pixel level adjustments or in an automated fashion using binarization software. The resulting enhanced images should have black ridges and white valleys. Pores and creases are also white. The resultant image should be life size and remain at the highest resolution available. Resizing the image should not use interpolation. If interpolation must be used to resize the image, a justification must be provided.

Reviewer: Stronger language, same meaning.

```suggestion
The image then needs to be processed such that the resultant image contains only black (0 in the 8-bit grayscale color space) and white (255 in the 8-bit grayscale color space) pixels. This can be done by hand using photo editing software suitable for pixel level adjustments or in an automated fashion using binarization software. The resulting enhanced images should have black ridges and white valleys. Pores and creases are also white. The resultant image should be life size and remain at the highest resolution available. Resizing the image shall not use interpolation without justification approved by the BIT.
```
